articles/what-api-endpoints-to-expose-to-the-public-internet.md

```diff
@@ -31,12 +31,15 @@ If you would like to use Fleet's MDM features, the following endpoints need to b
 - `/mdm/apple/scep` to allow hosts to obtain a SCEP certificate.
 - `/mdm/apple/mdm` to allow hosts to reach the server using the MDM protocol.
 - `/api/mdm/apple/enroll` to allow DEP-enrolled devices to get an enrollment profile.
-- `/api/*/fleet/device/*/mdm/apple/manual_enrollment_profile` to allow manually enrolled devices to
-  download an enrollment profile.
+- `/api/v1/fleet/device/*` to give end users access to their **My device** page. This page is where they download their manual enrollment profile, rotate their disk encryption key, and use other features. For more information on these API endpoints see the documentation [here](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/API-for-contributors.md#device-authenticated-routes).
```

Comment:
should we make a note that we need "For full Fleet Desktop and scripts functionality" as described under "Using Fleet Desktop on remote devices"?